### DISCUSSION TITLE
Recurring Order

### DISCUSSION THREAD

**Discussion by andris-sevcenko** [craftcms/commerce#2310] [Ideas]
> Created by: Jozo ([wmd@wmd.hr](wmd@wmd.hr)) on 2015/05/04 17:56:46 +0000
Votes at time of UserVoice import: 7

------
Or :) products that are  time based and they generate  automaticly offer/invoice for client  for next period

**Comment by nathansnelgrove** [comment]
This would be a game-changer for me, and I'd be able to use it on _so_ many client websites. This is a super common request among the NPOs I am frequently asked to work with.
